scripts/nordic-publish.ts

```diff
@@ -205,6 +210,20 @@ class ArtifactoryClient extends Client {
         return res.text();
     };
 
+    deleteCache = async (filename: string) => {
+        const url = `${this.cacheUrl}/${filename}`;
+        const res = await fetch(url, {
+            method: 'DELETE',
+            headers: { Authorization: `Bearer ${this.token}` },
+        });
+
+        if (!res.ok && res.status !== 404) {
```

Comment:
```suggestion
        if (!res.ok || res.status !== 404) {
```
not sure why not like this?